### DISCUSSION TITLE
Why not to move to GitLab?

### DISCUSSION THREAD

**Discussion by nidamanx** [netblue30/firejail#4456] [General]
**This sw is absolutely a great one.**
Why not to move to a real open platform like GilLab?
I think all community will approve and it will match better than to stay here.
Could be nice to think about.
Thanks to everione of you

**Comment by rusty-snake** [comment]
#2895

**Comment by reinerh** [comment]
> Why not to move to a real open platform like GilLab?

Gitlab uses an "open core" model, which makes it not a "real open platform".

A completely free/open platform would be something like Gitea. A popular Gitea provider for FOSS projects would for example be https://codeberg.org/.

**Comment by rusty-snake** [comment]
LOL, when did they started this?
![You must sign in to search for specific terms.](https://user-images.githubusercontent.com/41237666/133600941-8223332c-2ee5-4e8d-bab3-a27be3de11df.png)
GitLab is not only completely unusable if you disable javascript but also for non-registered users.

**Reply by reinerh** [reply]
It is (was?) also unusable if you want to use a name that ends like a file extension:
https://gitlab.com/gitlab-org/gitlab/-/issues/335278

**Reply by rusty-snake** [reply]
Right, this one too. And fyi it is fixed (deployed?). https://gitlab.com/gitlab-org/gitlab/-/merge_requests/65954/diffs

fefe about the fix (https://blog.fefe.de/?ts=9fbe3f7c):

> Der Fix ist jetzt, dass sie nach einem Punkt davor gucken. Wenn also jemand mit Nachnamen plain oder zip heißt, hat der immer noch verloren.

> The fix now is that they look for a dot in front of it. So if someone's last name is plain or zip, they still lose.

:rofl:

**Comment by crocket** [comment]
netblue30's own gitea instance would lead to more decentralization than gitlab.com which will eventually be bought by a big tech company in the future.

* firejail.xyz
* gitea.firejail.xyz
* ...

Running your own instance is like federation.
